implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -64,10 +78,11 @@ public static JWTAuthContextInfoProvider createWithKey(String publicKey, String
      * @return a new instance of JWTAuthContextInfoProvider
      */
     public static JWTAuthContextInfoProvider createWithKeyLocation(String publicKeyLocation, String issuer) {
-        return create(NONE, publicKeyLocation, issuer);
+        return create(NONE, publicKeyLocation, NONE, issuer);
     }
 
-    private static JWTAuthContextInfoProvider create(String publicKey, String publicKeyLocation, String issuer) {
+    private static JWTAuthContextInfoProvider create(String publicKey, String publicKeyLocation, String hmacSecret,
```

Comment:
IMHO we don't need this overload, as we can't have an option where both the asymmetric and secret keys are used